app/views/navigation/_desktop.html.haml

```diff
@@ -10,7 +10,8 @@
           DEV
     %ul.nav.navbar-nav.me-auto
       = nav_entry t("navigation.timeline"), root_path, icon: "home", hotkey: "g t"
-      = nav_entry t("navigation.inbox"), "/inbox", icon: "inbox", badge: inbox_count, badge_attr: { data: { controller: "pwa-badge" } }, hotkey: "g i"
+      #nav-inbox-desktop
```

Comment:
Please don't write invalid HTML. (first level element inside `ul/ol` needs to be `li` **always**)